articles/azure/azs-how-deploy-sql-template-powershell.md

```diff
@@ -146,47 +152,47 @@ The images used to create this deployment are:
-| DeploymentPrefix                | The DNS Prefix for the Public IP Address for the Always On Cluster            |                                                                       | aodns                                                                                |
-| DnsSuffix                       | The DNS Suffix for reverse lookup of public IPAddresses                       |                                                                       | azurestack.external                                                                  |
-| DomainName                      | The FQDN of the AD Domain created                                             |                                                                       | contoso.local                                                                        |
-| PlatformFaultDomainCount        | Fault Domain Count Integer                                                    |                                                                       | 1                                                                                    |
-| PlatformUpdateDomainCount       | Update Domain Count Integer                                                   |                                                                       | 1                                                                                    |
-| ScriptsBaseUrl                  | DSC Scripts base url                                                          |                                                                       | https<!---->://raw.githubusercontent.com/Azure/AzureStack-QuickStart-Templates/master/sq... |
-| SqlAOAGName                     | The Sql AlwaysOn Group Name                                                   |                                                                       | alwayson-ag                                                                          |
-| SqlAOListenerName               | The Sql AG Listener Name                                                      |                                                                       | \[tolower(concat('aon-listener-' , resourceGroup().name))\]                          |
+| ------------------------------- | ----------------------------------------------------------------------------- | --------------------------------------------------------------------- |------------------------------------------------------------------------------------ |
+| _artifactsLocation                   | The blob store where all deployment artifacts are stored          |                                                                       | https<!-- -->://raw.githubusercontent.com/Azure/AzureStack-QuickStart-Templates/master/sql-2016-alwayson
```

Comment:
Based on the parameters [here](https://github.com/Azure/AzureStack-QuickStart-Templates/tree/master/sql-2016-alwayson#parameters) it should 🤔. However, this may be a typo.

Reply:
https://github.com/Azure/AzureStack-QuickStart-Templates/blob/master/sql-2016-alwayson/azuredeploy.json#L6

Reply:
Looks like it needs to be there.